data-migrations/migrations/20240514162736-populate_origin_data_field_public.js.js

```diff
@@ -0,0 +1,46 @@
+export const up = async (db, client) => {
+  try {
+    const session = client.startSession();
+
+    await session.withTransaction(async () => {
+      const PublicCloudRequest = db.collection('PublicCloudRequest');
+
+      const publicCloudRequests = await PublicCloudRequest.aggregate([
+        {
+          $group: {
+            _id: '$licencePlate',
+            totalAmount: { $sum: 1 },
+          },
+        },
+      ]).toArray();
+
+      for (const requestsLicencePlate of publicCloudRequests) {
+        if (requestsLicencePlate.totalAmount > 1) {
+          const publicCloudRequestsLicencePlate = await PublicCloudRequest.find({
+            licencePlate: requestsLicencePlate._id,
+          })
+            .sort({ created: -1 })
```

Comment:
I believe `1` is `ASC`; means `old to new`.

Reply:
actually, we need the last one(`newest one`) request first, to assign it previous request `decisionDataId`. If it would be `old to new`, than we would need to try to the oldest request to assign next(`newer`) request `decisionDataId`